Makefile

```diff
@@ -116,7 +116,7 @@ install.podman: bin/conmon
 .PHONY: fmt
 fmt:
 	git ls-files -z \*.c \*.h | xargs -0 clang-format -i
-
+	git ls-files -z '*.bats' | xargs -0 sed -i 's/[[:space:]]\+$$//'
```

Comment:
should this be in this commit?

Reply:
This one is just missing format fix for BATS tests to avoid trailing spaces - intentional.